tests/PHPCurlClass/PHPCurlClassTest.php

```diff
@@ -1083,1193 +1075,91 @@ public function testJSONDecoder()
 
     public function testJSONContentTypeDetection()
     {
-        $json_content_types = array(
```

Comment:
Moved these two arrays (in a mildly modified manner) to the Helper class so they can be re-used without being re-defined.